FMSgridtools/shared/gridobj.py

```diff
@@ -1,20 +1,278 @@
+import dataclasses
+from typing import List, Optional
+
 import numpy as np
 import numpy.typing as npt
 import xarray as xr
-from typing import List, Optional, Type
-import dataclasses
 
+from gridtools.shared.gridtools_utils import check_file_is_there
```

Comment:
I think this directory name was changed after this PR was put in, should be `FMSgridtools.shared.gridtools_utils` now